base/applications/regedit/find.c

```diff
@@ -88,14 +88,22 @@ static BOOL CompareName(LPCWSTR pszName1, LPCWSTR pszName2)
 }
 
 static BOOL
-CompareData(
-    DWORD   dwType,
-    LPCWSTR psz1,
-    LPCWSTR psz2)
+CompareData(DWORD dwType, LPCVOID pv1, size_t cb1)
 {
-    INT i, cch1 = wcslen(psz1), cch2 = wcslen(psz2);
-    if (dwType == REG_SZ || dwType == REG_EXPAND_SZ)
+    if (dwType == REG_SZ || dwType == REG_EXPAND_SZ || dwType == REG_MULTI_SZ)
     {
+        LPCWSTR psz1 = pv1; /* We assume pv1 is UNICODE_NULL-terminated */
+        LPCWSTR psz2 = s_szFindWhat;
+        size_t cch1 = cb1 / sizeof(WCHAR), cch2 = wcslen(s_szFindWhat), i;
+
+#ifdef _DEBUG
```

Comment:
Regedit is not the right place for this. Ignore the extra byte and move on.

Reply:
Cc @tkreuzer

Reply:
Removed.